server/docker-compose.yml

```diff
@@ -1,13 +1,23 @@
-version: "3.8"
+version: '3.8'
 services:
   postgres:
     image: postgres:latest
     env_file: .env
     container_name: 'fairy-share-db'
     ports:
-      - "5555:5432"
+      - '5555:5432'
     volumes:
       - postgres_data:/var/lib/postgresql/data
 
+  postgres_test:
+    image: postgres:latest
+    env_file: .env.test
+    container_name: 'fairy-share-db-test'
+    ports:
+      - '5556:5432'
+    volumes:
+      - postgres_test_data:/var/lib/postgresql/data
+
```

Comment:
Now you start running DB container for dev and test. 
<img width="841" alt="Screenshot 2024-03-05 at 1 30 44 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/4d68a950-757a-409a-9706-bf5300e8ea7f">